src/Components/Publishing/Sections/__tests__/Sections.test.tsx

```diff
@@ -143,6 +144,21 @@ describe("Sections", () => {
         .instance() as any
       expect(wrapper.getContentEndIndex()).toBe(11)
     })
+
+    it("it injects display ads if feature when ads are enabled", () => {
+      props.article = FeatureArticle
+      props.isMobile = false
+      props.areHostedAdsEnabled = true
+      const wrapper = mountWrapper(props)
+      expect(wrapper.find(NewDisplayCanvas).length).toBe(2)
+    })
+
+    it("it does not inject display ads if feature when ads are disabled", () => {
+      props.article = FeatureArticle
+      props.isMobile = false
+      const wrapper = mountWrapper(props)
+      expect(wrapper.find(NewDisplayCanvas).length).toBe(0)
+    })
```

Comment:
Can we test that each of the correct ads are being inserted properly based on the relevant index? 

<img width="414" alt="Screen Shot 2019-06-06 at 12 40 52 PM" src="https://user-images.githubusercontent.com/236943/59061442-64229b80-8858-11e9-9135-baa8eb553ceb.png">

Reply:
Yes, that's a good idea